### PR TITLE
Add WifiMouse Server

### PR DESCRIPTION
**Pull request by s-crypt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16503)

### CODE REVIEW

**Comment by s-crypt** [comment]
The silent switch works, but because it is from an unknown publisher, there is an unskippable UAC prompt. Is there a workaround or is this a known limitation?

File: manifests/v/VividDesigns/WifiMouseServer/1.1/VividDesigns.WifiMouseServer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: VividDesigns.WifiMouseServer
+PackageVersion: 1.1
```
Comment (ItzLevvie) [inline manifests/v/VividDesigns/WifiMouseServer/1.1/VividDesigns.WifiMouseServer.installer.yaml:3]:
```suggestion
PackageVersion: 1.1.0.0
```

File: manifests/v/VividDesigns/WifiMouseServer/1.1/VividDesigns.WifiMouseServer.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: VividDesigns.WifiMouseServer
+PackageVersion: 1.1
```
Comment (ItzLevvie) [inline manifests/v/VividDesigns/WifiMouseServer/1.1/VividDesigns.WifiMouseServer.locale.en-US.yaml:3]:
```suggestion
PackageVersion: 1.1.0.0
```

File: manifests/v/VividDesigns/WifiMouseServer/1.1/VividDesigns.WifiMouseServer.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+PackageIdentifier: VividDesigns.WifiMouseServer
+PackageVersion: 1.1
```
Comment (ItzLevvie) [inline manifests/v/VividDesigns/WifiMouseServer/1.1/VividDesigns.WifiMouseServer.yaml:3]:
```suggestion
PackageVersion: 1.1.0.0
```

File: manifests/v/VividDesigns/WifiMouseServer/1.1/VividDesigns.WifiMouseServer.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+PackageLocale: en-US
+Publisher: Vivid Designs
```
Comment (ItzLevvie) [inline manifests/v/VividDesigns/WifiMouseServer/1.1/VividDesigns.WifiMouseServer.locale.en-US.yaml:5]:
```suggestion
Publisher: WifiMouseServer Authors
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see my review. Make sure to change the version folder.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**